backend/app/analysis/sentiment_analysis.py

```diff
@@ -0,0 +1,35 @@
+import nltk
+from nltk.sentiment import SentimentIntensityAnalyzer
+import numpy as np
+
+
+def text_to_sound(user_text):
+    """
+    Takes text, runs NLTK's sentiment analyzer, and then returns a sound object
+    """
+
+    nltk_analyzer = SentimentIntensityAnalyzer()
+    all_sentences = nltk.sent_tokenize(user_text)
+
+    all_scores = []
+    for each_sentence in all_sentences:
+        all_scores.append(nltk_analyzer.polarity_scores(each_sentence)["compound"] * 15)
+
+    all_frequencies = []
+    for each_score in all_scores:
+        frequency = 440 * (2 ** (each_score / 12))
```

Comment:
Same comment as above. One of the major goals of the current work is to leave things in a place where your fellow UROPs in the fall will be able to jump in and understand the context of what you all have built, so any clarify we can provide here on how and why we're using these values would be 🥇 .